aws/resource_aws_kinesis_firehose_delivery_stream_test.go

```diff
@@ -705,6 +760,14 @@ func TestAccAWSKinesisFirehoseDeliveryStream_ExtendedS3_ErrorOutputPrefix(t *tes
 					resource.TestCheckResourceAttr(resourceName, "extended_s3_configuration.0.error_output_prefix", "prefix2"),
 				),
 			},
+			{
+				Config: testAccKinesisFirehoseDeliveryStreamConfig_ExtendedS3_ErrorOutputPrefix(rName, rInt, ""),
```

Comment:
Test case for the bug I mentioned above where you can't unset the `ErrorOutputPrefix`